### PR TITLE
chore(breadcrumb): testing

### PR DESCRIPTION
**Pull request by alexandereneroth**
Function
- [x] Value picked up by form. (N/A)
- [x] Supports all native props. (N/A)
- [x] Change event is fired with the right name when not disabled. (N/A)
- [x] Change event is not fired when disabled. (N/A)
- [x] All props/methods/events/states are tested and works.

🆕Created ticket to remove disabled prop, as that is not needed.

- [x] It is possible to extract the values from input elements (N/A)

Appearance
- [x] Default state looks like in Figma.
- [x] Hover state looks like in Figma.
- [x] Active state looks like in Figma.
- [x] Focus state looks like in Figma.
- [x] Disabled state looks like in Figma.(N/A)
- [x] Component specific states looks like in Fimga.

Consistency
- [x] Props, methods, and events are consistent with other components

*Looks consistent to me*

### CODE REVIEW

File: src/components/Navigation/Header.tsx [hunk 1]
```diff
@@ -55,8 +56,14 @@ const Header = ({ pathname, toggleMobileNav }: HeaderProps) => {
...
-      <tds-header-item selected={pathname === "/text"}>
-        <Link to="text">Text</Link>
+      <tds-header-item
+        selected={pathname === "/lorem-ipsum/text-with-a-very-long-title"}
+      >
+        <Link to="lorem-ipsum/text-with-a-very-long-title">
+          <div className="header-item-text-wrap">
+            Text with a very long title
+          </div>
+        </Link>
```
Comment (mJarsater) [inline src/components/Navigation/Header.tsx:66]: What if this one was linked to "/lorem-ipsum", and that page then had a link in it to take you to "lorem-ipsum/text-with-a-very-long-title", which is not visible in the header. Then the breadcrumb would provide a more lifelike scenario maybe?

Comment (alexandereneroth) [reply src/components/Navigation/Header.tsx:66]: Good idea

File: src/index.css [hunk 2]
```diff
@@ -35,25 +35,26 @@ body,
...
+
+.wrapper {
   display: flex;
   flex: 1;
   flex-direction: column;
   gap: 32px;
   max-width: 600px;
   justify-content: center;
-  margin: 0 auto;
+  /* margin: 0 auto; */
```
Comment (mJarsater) [inline src/index.css]:
```suggestion
```

**Review by mJarsater — APPROVED** [review]

File: src/index.tsx [hunk 3]
```diff
@@ -1,90 +1,89 @@
...
         element: <WebComponents />,
       },
       {
-        path: "web-components",
+        path: 'web-components',
         element: <WebComponents />,
       },
       {
-        path: "table",
+        path: 'table',
```
Comment (theJohnnyMe) [inline src/index.tsx:33]: Not a part of this PR, but looks like we lost a navigation item for the table page, right?

File: src/utility.tsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const capitalizeFirst = (str: string) => {
```
Comment (theJohnnyMe) [inline src/utility.tsx:1]: Cool little helpers!

File: src/index.tsx (hunk 3 above)
Comment (mJarsater) [reply src/index.tsx:33]: I think it's the one just below this comment, right?

Comment (theJohnnyMe) [reply src/index.tsx:33]: I should be more specific: I meant in Header or Side menu. Route works as intended if you type it, but we have no navigation item in Header or Side menu to it.

Comment (mJarsater) [reply src/index.tsx:33]: Aah alright!

Comment (mJarsater) [reply src/index.tsx:33]: Yeah it looks like the side-menu is a bit off on mobile also. The banner is covering the close button. Maybe fix it in another PR?

**Review by theJohnnyMe — APPROVED** [review]
Great work Alex. Nice idea for using memo hook in React!
One small thing, if you feel like doing it:
In the `src/pages/WebComponents.tsx` breadcrumb wrapper on line 34 is singular but it should be "tds-breadcrumbs" - if that is fixed then the component is displayed correctly aka in one line...

File: src/index.tsx (hunk 3 above)
Comment (theJohnnyMe) [reply src/index.tsx:33]: Yep, another PR for sure, just notice it when going through this one  ;)

File: src/index.tsx [hunk 5]
```diff
@@ -1,51 +1,49 @@
...
+        path: 'form',
         element: <FormPage />,
       },
       {
-        path: "about",
+        path: 'about',
         element: <About />,
       },
       {
+        path: 'tabs-buttons',
```
Comment (alexandereneroth) [inline src/index.tsx:46]:
```suggestion
```

**Pull request merged**